src/main/java/org/fungover/haze/RedisServer.java

```diff
@@ -0,0 +1,28 @@
+package org.fungover.haze;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RedisServer {
```

Comment:
Since we are writing a Haze server, even if it talks RESP like Redis does we should probably not use that name on our classes?